core/block.go

```diff
@@ -35,12 +35,16 @@ type Header struct {
 	ProtocolVersion string
 	// Bloom filter on the events emitted this block
 	EventsBloom *bloom.BloomFilter
-	// Amount of WEI charged per Gas spent
-	GasPrice *felt.Felt
+	// Amount of WEI charged per Gas spent on L1
+	L1GasPriceETH *felt.Felt
+	// Amount of STRK charged per Gas spent on L2
+	L2GasPriceETH *felt.Felt
```

Comment:
Unfortunately it's not as simple as that. When we encode structures into bytes encoder stores field names alongside their values so if we rename it here we receive nil in these fields even if our db is up to date.

@IronGauntlets please confirm

Reply:
Can it be fixed with tags?

Reply:
I've checked on block №800:
```diff
        "l1_gas_price": {
            "price_in_fri": "0x0",
-           "price_in_wei": "0x476b2446"
+           "price_in_wei": null
        },
```
@kirugan you are right

Reply:
This commit fixes that.
a4761725237fc7c66d018410d45e389050d1ae55